src/libostree/ostree-gpg-verify-result.c

```diff
@@ -562,6 +562,14 @@ ostree_gpg_verify_result_describe_variant (GVariant *variant,
   key_id = (len > 16) ? fingerprint + len - 16 : fingerprint;
 
   date_time_utc = g_date_time_new_from_unix_utc (timestamp);
+  if (date_time_utc == NULL)
```

Comment:
It looks like in this case we'd hit at least one or two `g_return_if_fail()s`, and at least on glibc systems just end up printing `(null)`.  So, probably not exploitable.